src/EnergyPlus/IOFiles.hh

```diff
@@ -357,8 +360,20 @@ template <typename... Args> void print(std::ostream &os, fmt::string_view format
 
 template <typename... Args> void print(InputOutputFile &outputFile, fmt::string_view format_str, const Args &... args)
 {
-    assert(outputFile.os);
-    EnergyPlus::vprint(*outputFile.os, format_str, fmt::make_format_args(args...), sizeof...(Args));
+    auto *outputStream = [&]() -> std::ostream * {
+        if (outputFile.os) {
+            return outputFile.os.get();
+        } else {
+            if (outputFile.defaultToStdOut) {
+                return &std::cout;
+            } else {
+                assert(outputFile.os);
+                return nullptr;
+            }
+        }
+    }();
```

Comment:
I feel like this is a band-aid that we'll want to rip back off later.  It is probably indicative that the EDD writing code needs to be altered to write to `cout` instead.  But it'll work for now.  Thanks for getting it working.